### PR TITLE
Add player count to tournament signup section

### LINKED ISSUE #2597: Add a player count to the tournament screen
**Is your feature request related to a problem? Please describe.**
Often when I want to join a tournament, the maximum number of players has been reached, but I can't tell until I click the join button and it tells me I can't.

**Describe the solution you'd like**
It would be nice to have just a little player count on the tournament screen to see how many players have signed up!

### PR DESCRIPTION
**Pull request by cassidoo**
Fixes #2597

## Proposed Changes

  - Adds a player count to the tournament signup section

### CODE REVIEW

File: src/views/Tournament/Tournament.tsx [hunk 1]
```diff
@@ -1745,13 +1745,18 @@ export function Tournament(): JSX.Element {
...
+                            <>
+                                <Card>
+                                    {sorted_players.map((player) => (
+                                        <div key={player.id}>
+                                            <Player icon user={player} />
+                                        </div>
+                                    ))}
+                                </Card>
+                                <div className="player-count">
+                                    Number of players: {sorted_players.length}
```
Comment (anoek) [inline src/views/Tournament/Tournament.tsx:1757]: Need to translate this string, probably something like `{interpolate(_("Number of players: {{num_players}}"), { num_players: sorted_players.length})}`

**Review by anoek — CHANGES_REQUESTED** [review]
Sounds good to me, just need to update the text so it's translatable and we can merge

File: src/views/Tournament/Tournament.tsx (hunk 1 above)
Comment (cassidoo) [reply src/views/Tournament/Tournament.tsx]: Done!

**Comment by cassidoo** [comment]
I am not sure what the linter wants here, @anoek does it need the trailing comma there?

**Comment by anoek** [comment]
Yep, it wants the trailing comma.

FYI for future complaints like that you can run `npm run lint:fix` and it'll fix it for you. Also if you setup husky as per https://github.com/online-go/online-go.com/blob/devel/CONTRIBUTING.md it'll catch those before getting pushed up, and they show up in a more readable way.

**Review by anoek — APPROVED** [review]

**Pull request merged**